README.md

```diff
@@ -692,6 +692,10 @@ VS Code has a lot of nifty tricks and we try to preserve some of them:
 
   **Caveats:** This solution restores the default VS Code behavior for the <kbd>j</kbd> and <kbd>k</kbd> keys, so motions like `10j` [will not work](https://github.com/VSCodeVim/Vim/pull/3623#issuecomment-481473981). If you need these motions to work, [other, less performant options exist](https://github.com/VSCodeVim/Vim/issues/2924#issuecomment-476121848).
 
+- I've swapped Escape and Caps Lock with setxkbmap and it isn't working
```

Comment:
Can you elaborate on what is not working? Makes it easier for people to search.

Reply:
yes I've updated the wording